ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -305,13 +305,48 @@ public class BottomSheetController: UIViewController, Shadowable, TokenizedContr
     ///   - animated: Indicates if the change should be animated. The default value is `true`.
     ///   - completion: Closure to be called when the state change completes.
     @objc public func setIsHidden(_ isHidden: Bool, animated: Bool = true, completion: ((_ isFinished: Bool) -> Void)? = nil) {
-        let targetState: BottomSheetExpansionState = isHidden ? .hidden : .collapsed
+        if isHidden {
+            dismissSheet(animated: animated, completion: completion)
+        } else {
+            presentSheet(expandedState: .expanded, animated: animated, completion: completion)
+        }
+
+    }
+
+    /// Presents the bottom sheet view
+    /// - Parameters:
+    ///   - expandedState: The state the bottom sheet should expand to when presented
+    ///   - animated: Indicates if the change should be animated. The default value is `true`.
+    ///   - completion: Closure to be called when the state change completes.
+    @objc public func presentSheet(expandedState: BottomSheetExpansionState, animated: Bool = true, completion: ((_ isFinished: Bool) -> Void)? = nil) {
+        let finishedState: BottomSheetExpansionState
+        if expandedState != .collapsed && expandedState != .expanded {
```

Comment:
Minor: This seems like a good thing to convert to a switch/case statement so handling of each case is explicit and direct instead of implied? And enforces that any future cases get conscious thought

Reply:
Which would have side benefit of making this behavior clear to everyone. My first reaction was "why would we take the state passed in and ignore it in some cases?"